<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingTab</class>
 <widget class="QWidget" name="SettingTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="tournSaveLocation_2">
     <property name="text">
      <string>Tournament Save Location</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="tournSaveLocation"/>
     </item>
     <item>
      <widget class="QPushButton" name="tournSaveLocationButton">
       <property name="text">
        <string>Select Folder</string>
       </property>
       <property name="icon">
        <iconset theme="document-open"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="deckCountLabel">
     <property name="text">
      <string>Default Deck Count</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="deckCountSpinBox">
     <property name="suffix">
      <string> Decks</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="matchSizeLabel">
     <property name="text">
      <string>Default Match Size</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="matchSizeSpinBox">
     <property name="suffix">
      <string> Players</string>
     </property>
     <property name="minimum">
      <number>2</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="Line" name="line">
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="drawPointsLabel">
     <property name="text">
      <string>Default Draw Points</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="drawPointsSpinBox">
     <property name="suffix">
      <string> Points</string>
     </property>
     <property name="minimum">
      <number>-99</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="byePointsLabel">
     <property name="text">
      <string>Default Bye Points</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="byePointsSpinBox">
     <property name="suffix">
      <string> Points</string>
     </property>
     <property name="minimum">
      <number>-99</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="pointsLossLabel">
     <property name="text">
      <string>Default Loss Points</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QSpinBox" name="pointsLossSpinBox">
     <property name="suffix">
      <string> Points</string>
     </property>
     <property name="minimum">
      <number>-99</number>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="pointsWinLabel">
     <property name="text">
      <string>Default Win Points</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QSpinBox" name="pointsWinSpinBox">
     <property name="suffix">
      <string> Points</string>
     </property>
     <property name="minimum">
      <number>-99</number>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="16" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Reset|QDialogButtonBox::RestoreDefaults|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QRadioButton" name="radioButton">
     <property name="text">
      <string>Swiss</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QRadioButton" name="radioButton_2">
     <property name="text">
      <string>Fluid</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Default Tournament Type</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
